web/app/countrytopos.js

```diff
@@ -351,6 +351,13 @@ exports.addCountryTopos = function(countries) {
   countries['SS'] = getCountry('SSD')
   countries['ES'] = getSubUnits(['ESX', 'SEC', 'SEM']); //Spain Peninsula
   countries['ES-CN'] = getSubUnits(['ESC']); //Spain Canary Islands
+  // countries['ESC-LP'] = getCountry('La Palma')
```

Comment:
Can we keep the ISO names (https://en.wikipedia.org/wiki/ISO_3166-2:ES)?
If we subdivide at this level, I suggest having `ES-CN-...` in order to keep the prefix consistent.

Reply:
Sure I will switch the names to all have a ES-CN- prefix, happy with the second parts (FV/LP etc)?

Reply:
yep! all good